.env.example

```
@@ -0,0 +1,12 @@
+POSTGRES_DB="auth"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)